### PR TITLE
github: two more steps to release process

### PR DESCRIPTION
**Pull request by marquiz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ Please do not remove items from the checklist
 - [ ] Add a link to the tagged release in this issue.
 - [ ] Send an announcement email to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] node-feature-discovery $VERSION is released`
 - [ ] Add a link to the release announcement in this issue
+- [ ] For a major release, update README in master branch
+  - [ ] Update references to point to `$VERSION` and submit a PR
```
Comment (adrianchiris) [inline .github/ISSUE_TEMPLATE/new-release.md:42]: what are these references ?

should this be : run `prepare_release.sh $VERSION <KEY ID>` and only commit the README ?

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 2]
```diff
@@ -38,6 +38,10 @@ Please do not remove items from the checklist
 - [ ] Add a link to the tagged release in this issue.
 - [ ] Send an announcement email to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] node-feature-discovery $VERSION is released`
 - [ ] Add a link to the release announcement in this issue
+- [ ] For a major release, update README in master branch
```
Comment (adrianchiris) [inline .github/ISSUE_TEMPLATE/new-release.md:41]: what about a minor release ? e.g once v0.8.0 is released, say we need to release `v0.8.1` what will update helm chart in this case ?

File: .github/ISSUE_TEMPLATE/new-release.md (hunk 1 above)
Comment (marquiz) [reply .github/ISSUE_TEMPLATE/new-release.md:42]: > should this be : run `prepare_release.sh $VERSION <KEY ID>` and only commit the README ?
> 
> i assume it to replace any instance of current release with the new release e.g 0.7.0 -> 0.8.0

Basically yes, can be done with `prepare_release.sh`, too

File: .github/ISSUE_TEMPLATE/new-release.md (hunk 2 above)
Comment (marquiz) [reply .github/ISSUE_TEMPLATE/new-release.md:41]: Haven't had any, yet 🤞  But yes, a better solution would be nice

Comment (marquiz) [reply .github/ISSUE_TEMPLATE/new-release.md:41]: Found out that we can actually trigger on publish release event 🤦 
https://github.com/kubernetes-sigs/node-feature-discovery/pull/489

**Pull request merged**